package.json

```diff
@@ -31,7 +31,7 @@
     "scripts:lint:fix": "eslint --fix 'scripts/**' --ext .js,.jsx --env browser,node",
     "size": "npm run build && size-limit",
     "size:debug": "npm run build && size-limit --why",
-    "start": "node scripts/start.js",
+    "start": "npm run build && node scripts/start.js",
```

Comment:
hrm... How did this (or could this) work before, then?

Reply:
I'm not entirely sure. I was going to look into it more after this got merged.